### ISSUE TITLE
unable to play HLS/DASH uhd(4k) 

### ISSUE THREAD

**Issue by StigoSvK** [videojs/video.js#6458]
## Description
Hello everyone.
I have problem with playing uhd(4k) HLS or DASH VOD (without DRM). On android/ios native apps with different players implementation everything works. I normally init player and like always after receiving streaming data I pass src and type into player instance.

` player.src({
src: 'https://mydomain/xo-DN4YbGfSo8yPutl3vKA==,1582030616/vod/0050/4670/uhd-jldLyU4I/eng-uhd2_hevc.m3u8',
        type: 'application/x-mpegURL'
    });`

I'm able to run UHD stream from [https://github.com/videojs/videojs-contrib-hls/issues/1309](url) post when I pass `{hls: {bandwidth: 30000000}}})` to videojs options but it doesn't solve my problem 

Received m3u8 file:

> #EXTM3U
#EXT-X-VERSION:7
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="group_A1",NAME="audio_2",DEFAULT=YES,URI="media_2.m3u8"
#EXT-X-STREAM-INF:BANDWIDTH=10000091,RESOLUTION=3840x2160,AUDIO="group_A1"
media_1.m3u8

Received media_1.m3u8 file:

> #EXTM3U
#EXT-X-VERSION:6
#EXT-X-TARGETDURATION:4
#EXT-X-MEDIA-SEQUENCE:1
#EXT-X-MAP:URI="init_1.m4s"
#EXTINF:4.000000,
seg_1_000001.m4s
#EXTINF:4.000000,
seg_1_000002.m4s
#EXTINF:4.000000,
seg_1_000003.m4s
#EXTINF:4.000000,
seg_1_000004.m4s
#EXTINF:4.000000,
seg_1_000005.m4s
#EXTINF:4.000000,
...
seg_1_000986.m4s
#EXTINF:4.000000,
seg_1_000987.m4s
#EXTINF:0.920000,
seg_1_000988.m4s
#EXT-X-ENDLIST

Please do you have any advice?  I have similar issue with DASH. I'm pretty sure this problem its not caused by videojs player.  I don't understand why there are .m4s chunks because regular full hd and stream from post which I mentioned above have .ts chunks.

Thank you

## Results
![image](https://user-images.githubusercontent.com/28844291/74718209-07beab00-5232-11ea-9788-ec0c188ce977.png)

#### videojs
7.7.4 (without videojs-contrib-hls)
#### browsers
chrome, firefox

**Comment by ttshivers** [comment]
This issue is still present

**Comment by ttshivers** [comment]
This issue still exists

**Comment by gkatsev** [comment]
Is the stream only in HEVC or is it a mixed codec playlist? Most browsers do not support HEVC playback, even on mobile devices. VHS 2.1 which we just put into pre-release (and will go into Video.js soon) has better support for playback of mixed codec streams when the browser supports all codecs. It also fixes a whole bunch of bugs
Are you able to share a link to your stream? 4K streams do work but they may load slowly if they are h264 encoded (for example https://videojs-http-streaming.netlify.app/?debug=false&autoplay=false&muted=false&minified=false&liveui=true&partial=false&url=https%3A%2F%2Fd2zihajmogu5jn.cloudfront.net%2F4k-hls%2Fout.m3u8&type=application%2Fx-mpegurl&keysystems=)

**Comment by ttshivers** [comment]
The example stream on that page causes the error `Failed to execute 'appendBuffer' on 'SourceBuffer'` on latest chrome 84.0.4147.105 on Windows 10. (https://d2zihajmogu5jn.cloudfront.net/4k-hls/out.m3u8). If you play that stream from the start, the errors appear in the console and the video get stuck forever in buffering at around 30-40 seconds in.

**Comment by gkatsev** [comment]
Seems to mostly work fine for me, though. I occasionally see the sourcebuffer is full issue. We do plan on addressing the running out of source buffer soon.

**Comment by ttshivers** [comment]
Tested it again and issue is still present. Same procedure as my previous comment.

**Comment by znoooo** [comment]
Same error here, also if playlist 4K it's not present too (but original video encoded was 4K)

**Comment by znoooo** [comment]
Problem seems related to high value of bitarate on the video, forced bitarate <3000k and seems works

**Comment by gkatsev** [comment]
Yeah, high bitrates are the issue. We need to handle clearing the buffer better but haven't had the chance to focus on that yet.

**Comment by tolew1** [comment]
I'm unable to support 4k as of now because of this issue. Is there anything we can do or a work around? 4k usage is increasing and it would be great if this is supported.

**Comment by bastolino** [comment]
Stephan Hesse (@tchakabam) commented here https://github.com/videojs/videojs-contrib-hls/issues/705 that he had a patch against it under here: https://github.com/videojs/videojs-contrib-hls/pull/1242
I have no clue if the issue is really related in deep, but maybe it could help?
As this thing (not this issue, just to make sure nothing is misunderstood) seems to be in it for several years now, it was awesome if someone with more knowledge in the VideoJS-Code could have a look at it and maybe even hand in a PR?
I'd appreciate any fix, even a good patch :)

**Comment by Migacz85** [comment]
I'm using video.js 8 and still same bug. On egde there is sound no video. Chrome there is video for short time till it breaks and no sound. On android its working but instead of stereo it plays 5.1 sound 👯

**Comment by ChR-iSz** [comment]
Same here with video.js 8 and Wowza Streaming Server. Only audio with black video...
